ASUWebPlatforms/webspark-ci
# WS2 1823: Change UDS form styles to admin theme on admin-facing forms

### Description of problem:
UDS form styles display on admin-facing forms. Admin-facing forms should use the Admin theme.

### Solution:
The final solution was to modify form.inc to not add `uds-form` class to all forms, but to add the class only to webform forms via a webform template override.

[Travis' additions]
I have added styles to the non-uds forms to help provide consistency. I have tried hard to limit the styles to only styles in non-uds forms and layout-builder off-canvas dialogues for the most part. The only exception is fixing some of the items that we want to look like UDS, even though we're not using UDS anymore.

I paid close attention to the form validation styles. I felt that the required dot in layout-builder was too faint, so I darkened it and used the same colors for the validation errors. I think it makes it work a lot better. I also fixed it so that the underlines now apply to a few elements that didn't get that treatment previously.

New QA Steps: 

To test layout-builder-styles:

- When logged in as an admin, go to the Kitchen Sink page and click “Layout”
- Click into several of the blocks via pencil icon and “Configure”
- The “required” field SVG circle should now be a darker pink, so that it is more noticeable.
- Blank out a field that is required and then click the “Update” button at the bottom
- You should see an error notification at the top and the field that is empty should have the red border and the text below the field indicating that it needs to be populated
- Also, all of the other fields that pass validation should have the green border
- Re-populate that field and blank out a different required field. I recommend a required icon picker widget field or a ckeditor WYSIWYG field, since I added stylings for those. Then click “Update” again. The borders should update to match the current situation.

To test non-layout-builder styles:

- In Layout builder, add a new block to a random region, create a custom block, and select Hero.
- While adding media to the Hero block, choose to Add a new file, select a file from your computer, and without adding Alternative text and without checking the Decorative check box, hit Save.
- Verify that the Alternative text field border is red like the error text beneath the field, and that the check box and word Decorative (label) are grey (not green).
- Navigate to /webform and make sure that it displays with the UDS styles being applied. Use a similar approach to testing validation as described above. If you click into the Webform's pencil icon to edit its "Build" or other options, it should be using the Claro theme, so our theme changes should be irrelevant on the back-end.

### Links
- [JIRA ticket](https://asudev.jira.com/browse/WS2-1823)

### Checklist


### Verified in browsers


### Screenshots



Note: Sections that are not applicable for this issue can be removed.
